src/vs/editor/common/controller/oneCursor.ts

```diff
@@ -553,6 +552,18 @@ export class OneCursor {
 	public getViewLineMaxColumn(lineNumber:number): number {
 		return this.viewModelHelper.viewModel.getLineMaxColumn(lineNumber);
 	}
+	public getViewLineMinColumn(lineNumber:number): number {
+		return this.viewModelHelper.viewModel.getLineMinColumn(lineNumber);
+	}
+	public getViewLineCenterColumn(lineNumber:number): number {
```

Comment:
@alexandrudima I would be calling the constant for line center to be more specific - lineColumnCenter. This will give users clear understanding of what command does when this argument is used.